cli/cmd/main.go

```diff
@@ -74,91 +72,71 @@ func (c *imagesCmd) Run() error {
 
 type scanCmd struct {
 	JSONOutput bool     `short:"j" long:"json"   help:"Output in JSON format"`
-	Images     bool     `short:"i" long:"images" help:"Also output images for the target of each Earthfile (requires -t option)"`
 	Paths      []string `                        help:"paths to scan for Earthfiles"                                             arg:"" type:"path"`
-	Target     string   `short:"t"               help:"filter by Earthfiles that include this target"                                               default:""`
+	Target     []string `short:"t"               help:"filter by Earthfiles that include this target"                                               default:""`
```

Comment:
why do we need array here ?

Reply:
Yeah that's true, should be string

Reply:
My bad, it should be array since we can pass `-t target -t target2`